docs/source/lightning-module.rst

```diff
@@ -39,7 +39,7 @@ Notice a few things.
 
 |
 
-    .. code-block:: python
+    .. testcode:: python
```

Comment:
This change makes the docs check fails, as the `x.cuda()` does not work because the machine is not compiled with the cuda version of pytorch.

Reply:
oh yes, @SkafteNicki you are right. We need to keep this block as is.